scripts/manifest.js

```diff
@@ -44,10 +45,22 @@ const overrides = new Map([
 
 const icons = manifest.map(icon => {
 	const id = typeof icon === 'string' ? icon : icon.symbol;
+	const fileName = `${id.toLowerCase()}.svg`;
+	const svgPath = path.join(__dirname, '..', 'svg', 'color', fileName);
+	let color;
+	if (fs.existsSync(svgPath)) {
+		const svg = fs.readFileSync(svgPath, 'utf8');
+		const colors = getColors(svg);
+		const hex = colors.fills.map(color => color.hex().toUpperCase())[0];
```

Comment:
No need to map if you're just grabbing the first color, just grab it directly:

```js
colors.fills.[0].hex().toUpperCase();
```

Should probably also check it exists first.